gnuradio-runtime/lib/block.cc

```diff
@@ -739,9 +748,8 @@ namespace gr {
         basic_block_sptr blk = global_block_registry.block_lookup(block);
         blk->post(port, pmt::cons(pmt::mp("done"), pmt::mp(true)));
 
-        //std::cout << "notify finished --> ";
+        //GR_LOG_DEBUG (d_debug_logger, "notify finished -->");
```

Comment:
This was initially commented out. It means they are not at all used in current GNU Radio. Hence, I thought it was there for debugging purposes. Anyways, I will enable those.